CMakeLists.txt

```diff
@@ -52,6 +52,8 @@ if (SUPPORT_USAN AND WITH_USAN)
   set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fsanitize=undefined")
 endif()
 
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -rtlib=compiler-rt")
```

Comment:
Shouldn't we use this only on sanitizer build?

Reply:
I know I was testing it, it's not ready yet :smile: